src/commcare_cloud/ansible/roles/nginx/tasks/install.yml

```diff
@@ -94,6 +94,29 @@
   tags:
     - update-cert
 
+
+- name: Link CCHQ SSL cert
```

Comment:
Will these fail if the src file doesn't exist?

Reply:
It will create a broken symlink. But yeah that will fail the motive of the playbook if there is no source. (It assumes that source is managed by certbot cron. )